app/templates/components/as-calendar.hbs

```diff
@@ -1,5 +1,8 @@
 {{#if showHeader}}
-  {{as-calendar/header title=title model=model}}
+  {{as-calendar/header 
+    title=title 
+    model=model 
+    onNavigateWeek=(action "onNavigateWeek")}}
```

Comment:
I think here we could write `onNavigateWeek=attrs.onNavigateWeek` and get rid of the action in the as-calendar component :)